README.md

```diff
@@ -1,6 +1,20 @@
-# idena-pocket
-A web wallet for Idena.
+# Idena-pocket
```

Comment:
I'd like to maintain lowercase-dash-separated convention that is usually used in js package.